### ISSUE TITLE
Search: please add url and ID search in the JDA home page.

### ISSUE THREAD

**Issue by jdaKoko** [Japan-Digital-Archives/Japan-Digital-Archive#900]
Please add the url & ID search in the JDA home page search.  Adding the url & ID search makes the collection making processs a lot easier for the web harvesting team.

**Comment by corinnecurcie** [comment]
Hi Koko, do you mean that, by typing the URL or ID into the search box, you want the item from our database to appear in the search results? I think this is what you mean, I just want to be positive! thanks.

**Comment by jdaKoko** [comment]
Yes.  I am sorry it wasn't clear.  Thank you very much for working on  a lot of issues lately.  I am very grateful that the IT Crowd has been very helpful to our web harvesting team.
